project/plugins.sbt

```diff
@@ -11,4 +11,6 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.15.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.14.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.19")
+addSbtPlugin("uk.gov.hmrc" % "sbt-play-cross-compilation" % "0.12.0")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
```

Comment:
Since this is more an informational plugin rather than one that changes the build, maybe it is better to install it as a global plugin?

Reply:
Agreed. I'll take it out.